quickemu

```diff
@@ -528,7 +528,7 @@ function vm_boot() {
         case ${macos_release} in
         ventura|sonoma)
             if check_cpu_flag sse4_1 && check_cpu_flag avx2; then
-                CPU="-cpu Broadwell-noTSX-IBRS,kvm=on,vendor=GenuineIntel,+sse,+sse4.1,+avx,+avx2,+hypervisor,+popcnt,+aes,+xsave,+xsavec,+xsaveopt,+xgetbv1,+bmi1,+bmi2,+smep,+fma,+movbe,+invtsc"
+                CPU="-cpu Haswell,kvm=on,vendor=GenuineIntel,+sse3,+sse4.2,+aes,+xsave,+avx,+xsaveopt,+xsavec,+xgetbv1,+avx2,+bmi2,+smep,+bmi1,+fma,+movbe,+invtsc,+avx2"
```

Comment:
These commits don't make a whole lot of sense. Is there an issue that the CPU change @TuxVinyards did here caused? I think it indeed was a pointless change since all it really does is silence a couple of error messages on certain systems (not all, I still get errors on my 2 main systems, both of which have 7000 series Ryzen processors), but it shouldn't be reverted unless it causes a regression.

Similar can be said for some of the other commits you made at this time. Why, for example, did you remove macOS forcing an 'xhci' usb controller?

Reply:
The old CPU instruction had  `+sse4.2`  but grepped for 4.1.  It was messy and disorganised. Even had two entries for `+avx2` 

The new CPU  sse instruction is  now _correct_.   

I can't see that it hurts  to keep CPUID  functions a bit happier either.

Even if the qemu complaints were just general grumbling, it's still better to keep the screen a bit cleaner and have users a little more assured that things are okay.

I would like think that we could sort something out for Ryzen hardware too, in the future.

> but it shouldn't be reverted 

I'll  read that  as a "yes, okay"

> Why, for example, did you remove macOS forcing an 'xhci' usb controller?

If that was addressed to me,  it wasn't me.   I have only ever changed that one line. But @chasecovello  has forced it now in #981 so all is now fine, hopefully ....

Reply:
I wasn't commenting at all on your changes. They're fine, although I do feel they weren't the best use of time. @zen0bit had pushed a commit here reverting your change, and another one reverting the USB controller change. If the changes don't introduce a regression, there's no point reverting them.

Reply:
leftovers from merging...
is removed in #999